zebra/if_netlink.c

```diff
@@ -649,8 +649,6 @@ static int netlink_interface(struct sockaddr_nl *snl, struct nlmsghdr *h,
 	ifp = if_get_by_name(name, vrf_id, 0);
 	set_ifindex(ifp, ifi->ifi_index, zns);
 	ifp->flags = ifi->ifi_flags & 0x0000fffff;
-	if (IS_ZEBRA_IF_VRF(ifp))
```

Comment:
Why this code move?  There is no functional difference and no other return paths.

Reply:
zebra_if_set_ziftype() sets the zif_type (zebra interface type) which is used by the macro IS_ZEBRA_IF_VRF. Calling prior would not set VRF_LOOPBACK flag.